exchanges/coinut/coinut_wrapper.go

```diff
@@ -150,7 +150,7 @@ func (c *COINUT) Setup(exch *config.ExchangeConfig) error {
 		GenerateSubscriptions:            c.GenerateDefaultSubscriptions,
 		Features:                         &c.Features.Supports.WebsocketCapabilities,
 		OrderbookBufferLimit:             exch.WebsocketOrderbookBufferLimit,
-		BufferEnabled:                    true,
+		BufferEnabled:                    exch.WebsocketOrderbookBufferEnabled,
```

Comment:
getting rate limited on rest on my first try :/

Reply:
```
[INFO] |ORDERBOOK| 08/12/2020 16:44:03 |COINUT REST LTC-USDT SPOT: ORDERBOOK: Bids len: 200 Amount: 331.668272 LTC. Total value: 25603.5782255317 Asks len: 200 Amount: 167.338652 LTC. Total value: 14832.703570316102
```

Might be a rate-limiting issue, will write a note to revisit.